js/index.js

```diff
@@ -1 +1,62 @@
 // Your code goes here
+
+// PREVENTING NAV LINKS FROM RELOADING PAGE
+const nav = document.querySelector(".nav");
+
+nav.addEventListener('click', function(e) {
+    e.preventDefault();
+    console.log("nav link clicked");}
+    );
+
+
+// Flips the logo by clicking on it
+const navLogoHeading = document.querySelector('.logo-heading');
+
+navLogoHeading.addEventListener('dblclick', function (e) {
+    e.target.style.transform = "rotate(180deg)";});
```

Comment:
Lil challenge for ya: how could you have made this so it rotated 90 deg every dblclick? 😛